modules/core/src/lib/deck.js

```diff
@@ -766,6 +779,63 @@ export default class Deck {
       mode: 'hover'
     });
   }
+
+  _getFrameStats() {
+    this.stats.get('frameRate').timeEnd();
+    this.stats.get('frameRate').timeStart();
+
+    // Get individual stats from luma.gl so reset works
+    const animationLoopStats = this.animationLoop.stats;
+    this.stats
+      .get('GPU Time')
+      .addTime(
+        animationLoopStats.get('GPU Time') ? animationLoopStats.get('GPU Time').lastTiming : 0
+      );
+    this.stats
+      .get('CPU Time')
+      .addTime(
+        animationLoopStats.get('CPU Time') ? animationLoopStats.get('CPU Time').lastTiming : 0
+      );
+  }
+
+  /* eslint-disable complexity */
+  _getMetrics() {
+    this.metrics.fps = this.stats.get('frameRate') ? this.stats.get('frameRate').getHz() : 0;
+    this.metrics.setPropsTime = this.stats.get('setProps Time')
```

Comment:
I think `stats.get` always returns a `Stat` instance.

Reply:
You are absolutely correct. Forgot how my own API works :p